### DISCUSSION TITLE
Create a Latimer Core Example

### DISCUSSION THREAD

**Discussion by rondlg** [fieldmuseum/EMu-xml-to-json#13] [General]
Using the nearly complete Latimer Core Standard we should create an example here that can be linked to the non-normative documentation.

Question is what example dataset should we use?

**Comment by magpiedin** [comment]
Any accessions that are/can go into EMu+public & that haven't yet been catalogued?  (should we poll folks?)

**Comment by fmjjones** [comment]
Or one of the collection descriptions on the web https://docs.google.com/spreadsheets/d/1AXv3oBtXcBYP65qTgR6EiQvIduM53aF6v55x846ug0c/edit?usp=sharing

**Comment by magpiedin** [comment]
👍 - something like the [Mammal Collection](https://docs.google.com/spreadsheets/d/1AXv3oBtXcBYP65qTgR6EiQvIduM53aF6v55x846ug0c/edit#gid=1523278434&range=A16) / https://www.fieldmuseum.org/node/4831 ?

...in which case, dumb question -- how did we distinguish the 'real' grscicoll entries?
- https://www.gbif.org/grscicoll/collection/6999cd7a-b2a3-4f56-b5c4-0b0f1b70adba  vs  
- https://www.gbif.org/grscicoll/collection/6999cd7a-b2a3-4f56-b5c4-0b0f1b70adba ?

**Comment by fmjjones** [comment]
I was thinking more like https://www.fieldmuseum.org/node/18671
https://www.fieldmuseum.org/node/19506
https://www.fieldmuseum.org/node/4756
Where we could populate latimer core, map to EMu and Standards module. 3 birds and all that. I don't think those exist anywhere in EMu as developed collection descriptions. (Plus we could archive them.)
